canonical-web-and-design/maas-ui
# feat(ui): make LXD vm list reusable

## Done

- Update the LXD vms list components to pass a list of vms instead of a pod id.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the 'Virtual machines' tab for a LXD host.
- The table of VMs should load as it was before.

## Fixes

Fixes: canonical-web-and-design/app-squad#361.
